kebechet/managers/config_initializer/config_initializer.py

```diff
@@ -64,7 +65,7 @@ def run(self) -> typing.Optional[dict]:  # type: ignore
         thoth_config = pkg_resources.read_text(resources, "simple.thoth.yaml")
 
         with cloned_repo(self, depth=1, branch=self.project.default_branch) as repo:
-            prs = self.get_prs_by_branch(_BRANCH_NAME)
+            prs = self.get_prs_by_branch(_BRANCH_NAME, status=PRStatus.all)
```

Comment:
This would return closed pr along with open, is this something we want?
Is this part to check if pr already exists?

Reply:
Yeah, exactly. This was a small change which closes #1019